store/v2/root/store.go

```diff
@@ -272,6 +278,11 @@ func (s *Store) Commit(cs *corestore.Changeset) ([]byte, error) {
 		s.logger.Debug("commit header and version mismatch", "header_height", s.commitHeader.Height, "version", version)
 	}
 
+	// signal to the pruning manager that a new version has been committed
```

Comment:
```suggestion
	// signal to the pruning manager that a new version is about to be committed
    // this may be required if the implementation has an ongoing pruning process in the background which must be paused
```